content/en/docs/setup/_index.md

```diff
@@ -27,6 +27,11 @@ control, available resources, and expertise required to operate and manage a clu
 You can [download Kubernetes](/releases/download/) to deploy a Kubernetes cluster
 on a local machine, into the cloud, or for your own datacenter.
 
+[Kubernetes components](/docs/concepts/overview/components/) such as `API Server` or `Kube-proxy` can also be
```

Comment:
```suggestion
Several [Kubernetes components](/docs/concepts/overview/components/) such as `kube-apiserver` or `kube-proxy` can also be
```

Reply:
No worries this seems better I'll update these!